cw_core/lib/wallet_base.dart

```diff
@@ -67,6 +67,8 @@ abstract class WalletBase<BalanceType extends Balance, HistoryType extends Trans
 
   int calculateEstimatedFee(TransactionPriority priority, int? amount);
 
+  int calculateEstimatedFeeWithFeeRate(int feeRate, int? amount, {int? outputsCount, int? size});
```

Comment:
first, you can define this function with null so you don't have to add it to all other wallets that don't need it
i.e
```Dart
int calculateEstimatedFeeWithFeeRate(int feeRate, int? amount, {int? outputsCount, int? size}) => null;
```

and check if there is a better way to have this function without adding it to `wallet_base.dart`

Reply:
I've made it nullable.